themes/bootstrap5/js/get_this_dropdown.js

```diff
@@ -0,0 +1,121 @@
+let KEY_CODE = {
+  DOWN_ARROW: 40,
+  UP_ARROW: 38,
+};
+$(function pageLoad() {
+  let modal = document.querySelector('#modal');
+
+  /**
+   * Whether the dropdown is open
+   * @returns {boolean}
```

Comment:
Looks like some of the JSDoc is missing descriptions -- if you look at the files tab of the PR, you can see annotations for all of the warnings.